NAS-102996: Add nat_forwards field to docs (#1553)
freenas/freenas-docs: userguide/jails.rst

   |                        |              | addresses, enter the host MAC address and the jail MAC address separated by a space.                    |
   |                        |              |                                                                                                         |
   +------------------------+--------------+---------------------------------------------------------------------------------------------------------+

>    | nat_forwards           | checkbox     | `Network Address Translation (NAT) port forwarding <https://en.wikipedia.org/wiki/Port_forwarding>`__   |

Reviewer: Shouldn't this be _NAT Forwards_, or are we still trying to stay with the same names as iocage fields?

Author: This is what it is in the UI right now. I don't doc what it isn't

>    |                        |              | options. Setting for :guilabel:`NAT` jails adds :guilabel:`protocol`, :guilabel:`jail_port`, and        |

Reviewer: s/Setting for/Enabling on/

   |                        |              | :guilabel:`host_port` port forwarding options. Additional port forwarding configurations can be created |

>    |                        |              | by clicking |ui-add|. Multiple jails can set the same :guilabel:`jail_port` value, but only             |

Reviewer: ```suggestion
   |                        |              | by clicking |ui-add|. Multiple jails can be set to use the same :guilabel:`jail_port`, but only         |
```

>    |                        |              | one jail can be running with this configuration.                                                        |

Reviewer: ```suggestion
   |                        |              | one running jail can use an individual port.                                                        |
```
